code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -90,6 +89,22 @@
 	body_color = "gray"
 	icon_state = "mouse_gray"
 
+/mob/living/simple_animal/mouse/mouse_op
+	name = "mouse operative"
+	minbodytemp = 0
+	maxbodytemp = 5000
+	maxHealth = 20
+	health = 20
+	mob_biotypes = list(MOB_INORGANIC)
+	desc = "Oh no..."
+	icon = 'yogstation/icons/mob/animal.dmi'
+	icon_state = "mouse_operative"
+
+/mob/living/simple_animal/mouse/mouse_op/death(var/gibbed = FALSE, toast)
+	. = ..()
+	if(gibbed == FALSE)
+		src.gib()
```

Comment:
you have problem with the original death proc that will produce a mouse corpse and qdel itself before you try to gib it